nifxml.py

```diff
@@ -719,7 +755,7 @@ def member_name(n):
     @rtype: string
     """
     if n == None: return None
-    if n == 'ARG': return 'ARG'
+    if n.startswith('ARG'): return n
```

Comment:
I have not yet examined all the functions of gen_niflib, so I copied the code of the figment without analysis. In addition, I'm using the python for the first time, and I can not quickly understand each line of code.